File: src/main/java/com/slackow/func/parser/value/values/BooleanValue.java
package com.slackow.func.parser.value.values;

import com.slackow.func.parser.value.Value;
import com.slackow.func.parser.value.types.BooleanType;

public class BooleanValue extends Value {

    private boolean value;

    public BooleanValue(Boolean b) {
        super(BooleanType.INSTANCE);
    }

    @Override
    public String toJsonValue() {
        return toString();
    }
}

File: src/main/java/com/slackow/func/parser/value/values/BooleanValue.kt
package com.slackow.func.parser.value.values

import com.slackow.func.parser.value.Type
import com.slackow.func.parser.value.Value
import com.slackow.func.parser.value.types.BooleanType

class BooleanValue(val value: Boolean?) : Value(BooleanType) {

    override fun toJsonValue(): String {
        return toString()
    }
}